### PR TITLE
Fix/refactor code

### PR DESCRIPTION
**Pull request by chandreshkkhatri**
- refactor request validation code to use common middleware and standardize Joi schemas  
- refactor test cases to allow passing them, comment out redundant test cases

### CODE REVIEW

File: config/test.json
Comment (shital01) [inline config/test.json:1]: ye templateIdDebit wale use ho rhe h kya testing script me ,waise jab apn local run karte h toh sms ya notification nahin jate server pe deploy karte h tabhi work karte h woh

Comment (chandreshkkhatri) [reply config/test.json:1]: haan but error throw kar rhe thhe test cases so i added them

File: utils/validations/uploadUrlRequestValidations.js
Comment (shital01) [inline utils/validations/uploadUrlRequestValidations.js:1]: upload request schema wrong copy hua h ,kindly check

Comment (chandreshkkhatri) [reply utils/validations/uploadUrlRequestValidations.js:1]: update with fix, thanks

**Pull request merged**